extensions/dict_dcca_3gpp/dict_dcca_3gpp.c

```diff
@@ -10887,6 +10905,17 @@ static int dict_dcca_3gpp_entry(char * conffile)
 			};
 			PARSE_loc_rules(rules, rule_avp);
 	}
+	{
+		struct dict_avp_data data = {
+			458,	/* Code */
+			13019,	/* Vendor */
+			"Reservation-Priority",	/* Name */
+			AVP_FLAG_VENDOR |AVP_FLAG_MANDATORY,	/* Fixed flags */
+			AVP_FLAG_VENDOR,	/* Fixed flag values */
+			AVP_TYPE_UNSIGNED32	/* base type of data */ /* XXX: guessed */
```

Comment:
Yes, it's enumerated so `UNSIGNED32` is correct.  Do you also need to define the enumeration?